### ISSUE TITLE
Import treined models object detection on tensorflow python to tfjs

### ISSUE THREAD

**Issue by DarkNavrel** [tensorflow/tfjs#1392]
Tensorflow.js Object Detection
Hello everyone, I'm starting now in tfjs but I already had a familiarity with the Tensorflow framework since version 0.9 for python, my question is: what is the simplest way to import a trained model with Tensorflow Python to one that can be used in tfjs? I have already converted frozen_graph.pb and got a folder with the model.json, weights and 7 binary files. In the examples I found as a reference like coco_ssd and mobilenet the models are directly imported from the web, could you help me with some code that imports the files generated directly from the conversion please? (This is to directly import the binary files, the model.json and the file containing the weights).

Sorry if I'm being a complete idiot but really that part was not clear to me.

**Comment by tafsiri** [comment]
Have you tried the steps here https://www.tensorflow.org/js/tutorials/conversion/import_saved_model? (in particular step 2 on that page).

Let us know if you run into any trouble.

**Comment by DarkNavrel** [comment]
Yes, I did exactly as described in:

tensorflowjs_converter \
     --input_format = tf_frozen_model \
     --output_names = 'Softmax' \ * Softmax was my output layer
     /mobilenet/frozen_model.pb \
     / mobilenet / web_model

And I got a folder exactly with the 9 files:
group1-shard1of7 ...
group1-shard7of7
tensorflowjs_model.pb
weights_manifest.json

I did not get the model.json file in the conversion to code:

import * as tf from '@ tensorflow / tfjs';
import {loadGraphModel} from '@ tensorflow / tfjs-converter';

const MODEL_URL = 'model_directory / model.json';

const model = await loadGraphModel (MODEL_URL);
const cat = document.getElementById ('cat');
model.execute (tf.fromPixels (cat));

Was there a conversion error that did not generate model.json? Or should I get it any other way?

**Comment by vpanichkin** [comment]
Agree with @DarkNavrel, also this tutorial cannot be used for updated tensorflowjs_converter, as frozen_model is deprecated

**Comment by pyu10055** [comment]
Object detection models can export saved model, please use the latest converter against the exported saved model format for the retrained model.
